src/main/java/uk/gov/companieshouse/web/pps/util/PenaltyUtils.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.companieshouse.web.pps.util;
+
+public class PenaltyUtils {
+
+    private PenaltyUtils() {
+    }
+
+    public static String getPenaltyReason(final String penaltyReference) {
+        return penaltyReference.startsWith("PN") ? "confirmation statement" : "Late filing of accounts (3-6 months after due date)";
```

Comment:
Please can you look to move these hard coded reasons into our `src/main/resources/application.properties`?
The reason for LFP should be set to `Late filing of accounts`.
We could have something like `penalty.view-penalties-late-filing-reason=Late filing of accounts`